app/scripts/controllers/eventDetails.js

```diff
@@ -25,10 +25,6 @@ angular.module('confRegistrationWebApp')
     };
     $scope.changeTab($scope.tabs[0]);
 
-    $scope.paymentGateways = [
```

Comment:
It is only used to generate the "Payment Gateway" selector that I removed in this PR.